setup.py

```diff
@@ -55,24 +65,21 @@
     name='elixir-daisy',
     version='1.6.0',
     description="Elixir-LU DAISY",
-    author="Pinar Alper, Valentin Grouès, Yohan Jarosz, Jacek Lebioda, Kavita Rege",
-    author_email='valentin.groues@uni.lu',
+    author="Pinar Alper, Valentin Grouès, Yohan Jarosz, Jacek Lebioda, Kavita Rege, Vilem Ded",
+    author_email='lcsb.sysadmins@uni.lu',
     url='https://github.com/elixir-luxembourg/daisy',
     packages=find_packages(exclude=['contrib', 'docs', 'tests*']),
     package_dir={'elixir_daisy':
                      'elixir_daisy'},
     include_package_data=True,
     install_requires=requirements,
+    dependency_links=["https://github.com/jLebioda/mockldap2.git"],
```

Comment:
shouldn't this be the elixir-luxembourg repo?

Reply:
this should be actually gone :) Just corrected it, thanks!